### PR TITLE
Reset Center Align & Reposition option

### PR DESCRIPTION
**Pull request by plug-n-play**
* [ ]  Passed [sanity tests](http://bit.ly/sttm-sanity-tests).
* [ ]  Ran `npm test` & fixed newly introduced lint errors.
* [ ]  Checked console for errors.

<!-- New to markdown? Simply put an 'x' between [ ] to check it. Like [x] this. (No spaces).

### CODE REVIEW

File: src/js/features/reducers/index.js [hunk 1]
```diff
@@ -230,7 +232,22 @@ export default function reducer(state, action) {
...
+        centerAlignGurbani,
+      };
+    }
+    case SET_CENTER_ALIGN_OPTION: {
+      const centerAlignGurbani = action.payload;
+
+      saveToLocalStorage(
+        LOCAL_STORAGE_KEY_FOR_CENTER_ALIGN_VIEW,
+        centerAlignGurbani
+      );
```
Comment (bogas04) [inline src/js/features/reducers/index.js]: What's the difference between toggle and set?

Comment (plug-n-play) [reply src/js/features/reducers/index.js]: Toggle - when user toggles the option. Set, used when user clicks reset. Followed the same pattern as used in `TOGGLE_DARK_MODE` `SET_DARK_MODE`

Comment (tarunsingh5) [reply src/js/features/reducers/index.js]: @bogas04 is there any action needed here?

File: src/js/features/actions/index.js [hunk 2]
```diff
@@ -7,6 +7,7 @@ import {
   DEFAULT_FONT_FAMILY,
   DEFAULT_CENTER_ALIGN_GURBANI,
 } from '../../constants';
+import store from '../../features/store';
```
Comment (bogas04) [inline src/js/features/actions/index.js]: I think you can use `@/features/stores`

File: src/js/features/actions/index.js [hunk 3]
```diff
@@ -87,5 +88,7 @@ export const resetFontOptions = () => dispatch => {
 export const SET_CENTER_ALIGN_OPTION = 'SET_CENTER_ALIGN_OPTION';
 export const setCenterAlignOption = createAction(SET_CENTER_ALIGN_OPTION);
 
-export const TOGGLE_CENTER_ALIGN_OPTION = 'TOGGLE_CENTER_ALIGN_OPTION';
-export const toggleCenterAlignOption = createAction(TOGGLE_CENTER_ALIGN_OPTION);
+export const toggleCenterAlignOption = () => dispatch => {
+  const state = store.getState();
+  dispatch(setCenterAlignOption(!state.centerAlignGurbani));
+};
```
Comment (bogas04) [inline src/js/features/actions/index.js]:
```suggestion
export const toggleCenterAlignOption = () => (dispatch, getState) => {
  const state = getState();
  dispatch(setCenterAlignOption(!state.centerAlignGurbani));
};
```

**Comment by bogas04** [comment]
Since this is to fix center align related issues, could you also make this change;

Remove Center Align from the search page, as it's useless there.

**Pull request merged**